commands/command_clean.go

```diff
@@ -22,7 +23,7 @@ import (
 //
 // If the object read from "from" is _already_ a clean pointer, then it will be
 // written out verbatim to "to", without trying to make it a pointer again.
-func clean(to io.Writer, from io.Reader, fileName string, fileSize int64) error {
+func clean(to io.Writer, from io.Reader, fileName string, fileSize int64) (nr, nw int64, err error) {
```

Comment:
I agree that these are confusing. I rewrote this in a way that those return values aren't needed, instead `commands.clean()` returns an `(*lfs.Pointer, error)`, which we can access the `Size` member of.